docusaurus.config.js

```diff
@@ -71,72 +57,22 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
-      // Replace with your project's social card
-      image: 'img/docusaurus-social-card.jpg',
       navbar: {
-        title: 'My Site',
+        title: 'Portals Knowledgebase',
         logo: {
-          alt: 'My Site Logo',
+          alt: 'Skills Network Logo',
           src: 'img/logo.svg',
         },
         items: [
           {
-            type: 'docSidebar',
-            sidebarId: 'tutorialSidebar',
-            position: 'left',
-            label: 'Tutorial',
-          },
-          { to: '/blog', label: 'Blog', position: 'left' },
-          {
-            href: 'https://github.com/facebook/docusaurus',
-            label: 'GitHub',
+            href: 'https://portals.skills.network',
+            label: 'Portal Manager',
```

Comment:
Portal Manager is an internal service for our team so no need to link to it here